.babelrc

```diff
@@ -8,16 +8,5 @@
     "@babel/plugin-proposal-object-rest-spread",
     "@babel/plugin-proposal-class-properties",
     "@babel/plugin-transform-runtime"
-  ],
-  "env": {
-    "test": {
-      "presets": [
-        ["./babel", {
-          "preset-env": {
```

Comment:
Since we use webpack for the server, this should work great.
Earlier we had to recompile import,export syntax to commonjs. That's why we use this.

Reply:
Are you saying we could just use `next/babel` 🤔 I believe that doesn't work since the tests themselves use `import/export` and are not compiled by babel 🕵️